cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/JSONParser.scala

```diff
@@ -263,7 +285,8 @@ object JSONParser {
   implicit val exeOptionReads: Reads[QueryExeOption] = (
     (__ \ QueryExeOption.TagSliceMillis).readNullable[Int].map(_.getOrElse(-1)) and
       (__ \ QueryExeOption.TagContinueSeconds).readNullable[Int].map(_.getOrElse(-1)) and
-      (__ \ QueryExeOption.TagLimit).readNullable[Int].map(_.getOrElse(Int.MaxValue))
+      // this "limit" value will be discarded, real limit value is parsed from request
```

Comment:
so we should remove it?

Reply:
Unless we have a case class that contains only `sliceMillis` and `ContinueSeconds`. This is why last time I create another whatever case class to parse it.

Reply:
Got it.